tests/pages/firefox/new/sticky_promo.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import BasePage
+from pages.regions.sticky_promo import StickyPromo
+
+
+class StickyPromoPage(BasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/new/'
+
+    _sticky_promo_modal_content_locator = (By.CSS_SELECTOR, '.mzp-c-sticky-promo.mzp-a-slide-in')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda s: self.seed_url in s.current_url)
+        promo = self.find_element(*self._sticky_promo_modal_content_locator)
```

Comment:
Not sure when this was removed, but I'd suggest still waiting for the `loaded` class on the HTML element before running the tests (in addition to what you already have here): https://github.com/mozilla/bedrock/blob/master/tests/pages/base.py#L32-L33